src/controllers/controller.line.js

```diff
@@ -117,18 +117,16 @@ module.exports = DatasetController.extend({
 		var me = this;
 		var meta = me.getMeta();
 		var custom = point.custom || {};
-		var dataset = me.getDataset();
 		var datasetIndex = me.index;
-		var value = dataset.data[index];
 		var xScale = me._xScale;
 		var yScale = me._yScale;
 		var lineModel = meta.dataset._model;
 		var x, y;
 
 		var options = me._resolveDataElementOptions(point, index);
 
-		x = xScale.getPixelForValue(typeof value === 'object' ? value : NaN, index, datasetIndex);
-		y = reset ? yScale.getBasePixel() : me.calculatePointY(value, index, datasetIndex);
+		x = xScale.getPixelForValue(me._getParsedValue(xScale, index));
+		y = reset ? yScale.getBasePixel() : yScale.getPixelForValue(me._getStackedValue(yScale, index));
```

Comment:
I think its worth the extra function call to separate internal data storage behind API.
But maybe it would be better to get both, x/y in same call.